src/js/main-background.js

```diff
@@ -319,6 +320,25 @@ window.scroblrGlobal = (function () {
                 }, 5000);
             }
         }
+        
+        // Chrome 28+ notifications
+        if (chrome && getOptionStatus("notifications")) {
```

Comment:
Oh, good point. For some time webkitNotifications and chrome.notifications was both present in Chrome, so yes, if someone have Chrome version which supports both, then yes, 2 notifications will be sent. Going to adjust the patch then.